client/lib/domains/constants.js

```diff
@@ -17,7 +17,40 @@ const registrar = {
 	MAINTENANCE: 'Registrar TLD Maintenance'
 };
 
+const tlds = {
```

Comment:
Hmm right I think the next step is to remove those product_slug when all the gTLDs come in. So in that regard both solution will need an update. In the end, they should be equivalent maintenance-wise, we'll need to search the whole repository to update instances of domain_reg.
I'll keep the current solution because it has the merit of being there and working. @yurynix if you feel strongly about that I suggest we do this in another PR